packages/cardpay-sdk/sdk/utils/conversions.ts

```diff
@@ -63,3 +68,14 @@ export async function getCurrentGasPrice(chainId: number): Promise<GasPrice> {
     fast: BigNumber.from(gasPriceJson.fast),
   };
 }
+
+export async function getNativeWeiInToken(provider: JsonRpcProvider, tokenAddress: string): Promise<BigNumber> {
```

Comment:
I am confused with the combination of this function's name and its implementation.  The name suggests it converts the native coin value in wei to the specified token, but there is a call inside to get the gas price. These are two different things and I feel this can be simplified or renamed for better clarity.

Reply:
updated